src/components/AutoAssignDialog.tsx

```diff
@@ -28,6 +27,12 @@ enum SelectionType {
   "Selected" = 1,
 }
 
+enum AssigneesType {
+  "EntireTeam" = 0,
+  "MembersAndCollaborators" = 1,
+  "Collaborators" = 2,
+}
+
```

Comment:
minor but I think we can just do:

```suggestion
enum AssigneesType {
  "EntireTeam",
  "MembersAndCollaborators",
  "Collaborators",
}
``` 

here as we don't need specific values for the enums?